tutorial/chapter/3/index.md

```diff
@@ -125,32 +115,51 @@ function(x, y) {
 }
 ```
 
-JavaScript's usual relaxed rules around argument count can lead to extremely
-subtle and hard to detect bugs. It's potentially more flexible the JavaScript
-way, but much more error-prone.
+JavaScript's usual relaxed rules around argument count can lead to extremely subtle and hard to detect bugs. It's potentially more flexible the JavaScript way, but much more error-prone.
+
+Sometimes APIs are much easier if they can have varying parameter counts, so it is still possible in Squiggle. Using `...` in front of your last parameter causes it so collect all of the extra parameters in an array.
 
-Sometimes APIs are much easier if they can have varying parameter counts, so it
-is still possible in Squiggle. Using `...` in front of your last parameter
-causes it so collect all of the extra parameters in an array.
+### Multi-statement functions
+
+Functions may have multiple statements in them if you use this form with `do/end`:
```

Comment:
I’d phrase this as “if you use the `do/end` form”, but maybe I’m too accustomed to Lisp form documentation. :stuck_out_tongue: